use_cases/eluc/predictors/evaluation/validator.py

```diff
@@ -0,0 +1,42 @@
+"""
+Validation of input and output dataframes for predictor evaluation.
+"""
+import pandas as pd
+
+class Validator():
+    """
+    Validates input and output dataframes for predictor evaluation.
+    Context, actions, outcomes do not necessarily have to match the project's CAO_MAPPING. For example, if we are
+    just evaluating ELUC we can just pass the single column as outcomes.
```

Comment:
But we still need to have *at least* the context and actions in the input of the predictor, right?

Reply:
Have you looked at https://github.com/cognizant-ai-labs/covid-xprize/blob/master/covid_xprize/validation/predictor_validation.py for inspiration?

Reply:
> But we still need to have _at least_ the context and actions in the input of the predictor, right?

Currently we leave it open to the Validator which context/actions to make sure are in the inputs. When we create the Validator in the Scoring script we pass in our correct CA and just ELUC as O.